diff --git a/src/extensions/ostro/ostro-access-control-manager.c b/src/extensions/ostro/ostro-access-control-manager.c
index 2324c34..1e3c332 100644
--- a/src/extensions/ostro/ostro-access-control-manager.c
+++ b/src/extensions/ostro/ostro-access-control-manager.c
@@ -167,7 +167,7 @@ extension_ostro_access_control_manager_peer_is_allowed_to_use_identity (
                             GSignondAccessControlManager *self,
                             const GSignondSecurityContext *peer_ctx,
                             const GSignondSecurityContext *identity_owner,
-                            const GSignondSecurityContextList *identity_acl)
+                            const GList *identity_acl)
 {
     GSignondSecurityContext* acl_ctx = NULL;
     const gchar *peer_system_ctx = gsignond_security_context_get_system_context(peer_ctx);
diff --git a/src/extensions/ostro/ostro-access-control-manager.h b/src/extensions/ostro/ostro-access-control-manager.h
index 30e4e72..4663d1a 100644
--- a/src/extensions/ostro/ostro-access-control-manager.h
+++ b/src/extensions/ostro/ostro-access-control-manager.h
@@ -85,7 +85,7 @@ extension_ostro_access_control_manager_peer_is_allowed_to_use_identity (
                             GSignondAccessControlManager *self,
                             const GSignondSecurityContext *peer_ctx,
                             const GSignondSecurityContext *identity_owner,
-                            const GSignondSecurityContextList *identity_acl);
+                            const GList *identity_acl);
 
 gboolean
 extension_ostro_access_control_manager_peer_is_owner_of_identity (
diff --git a/src/extensions/tizen/tizen-access-control-manager.c b/src/extensions/tizen/tizen-access-control-manager.c
index 355f09f..ed911ae 100644
--- a/src/extensions/tizen/tizen-access-control-manager.c
+++ b/src/extensions/tizen/tizen-access-control-manager.c
@@ -163,7 +163,7 @@ extension_tizen_access_control_manager_peer_is_allowed_to_use_identity (
                             GSignondAccessControlManager *self,
                             const GSignondSecurityContext *peer_ctx,
                             const GSignondSecurityContext *identity_owner,
-                            const GSignondSecurityContextList *identity_acl)
+                            const GList *identity_acl)
 {
     GSignondSecurityContext* acl_ctx = NULL;
     const gchar *peer_system_ctx = gsignond_security_context_get_system_context(peer_ctx);
